.goreleaser.yml

```diff
@@ -94,7 +97,8 @@ blobs:
     folder: "{{ .ProjectName }}"
 
 dockers:
-  - goos: linux
+  - id: amd64image
+    goos: linux
```

Comment:
this is in prep for manifest build. However - buildx is only available via docker desktop? so I can't get any further.

Reply:
we should probably add a note in the readme that we're only building amd64 docker images currently

Reply:
sure. its building your local platform. we need to build a manifest-build and specify both platforms for it to work properly.

Reply:
>sure. its building your local platform. we need to build a manifest-build and specify both platforms for it to work properly for both. That requires buildx.

To clarify for myself, `colima` is able to process the docker image goreleaser is buildiong because it uses my local machine's platform, but for a manifest build, we would want both amd64 and arm64 to be produced and that requires `buildx`?

Reply:
exactly so

Reply:
I've added a note about the OS and arch for the docker build to the README.